pootle/apps/pootle_project/models.py

```diff
@@ -30,11 +30,12 @@
 from pootle.core.cache import make_method_key
 from pootle.core.mixins import CachedTreeItem
 from pootle.core.models import VirtualResource
-from pootle.core.url_helpers import (get_editor_filter, get_path_sortkey,
-                                     split_pootle_path, to_tp_relative_path)
```

Comment:
I have to disagree here, anyone that reads the method/docstring will have a clue of what it's supposed to do versus trying to guess the meaning of the split. The helper is meant to hide the *how* and show the *what*, i.e show the intention explicitly.

Remember not everyone is familiar with the code :wink: — that's IMHO the reason why this helps and the motivation not to remove it.